DrewBradley/rancid-tomatillos
# Feature/additional css styling

### What does this PR do?
- This PR adds CSS styling and spacing to some of the individual movie details.

### What does it fix?
- Nothing to fix at this point.

### Is this a feature or a fix?
- This is adding to an existing feature, styling our individual movie view.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start.

### How should this be tested?
- Nothing to test, but you can click on a thumbnail to see the additional styling in the footer component of our individual view.
